data_driven_onboarding/schema.py

```diff
@@ -165,9 +165,27 @@ class DDOSuggestionsResult(graphene.ObjectType):
     )
 
     stabilized_unit_count_2017 = graphene.Int(
-        required=True,
         description=(
             "The number of rent-stabilized residential units at the BBL in 2017."
+        ),
+        deprecation_reason=(
+            "This field has been deprecated as we now use `stabilized_unit_count` "
+            "to store the rs unit count for the most up-to-date-year we have available."
+        )
```

Comment:
NOICE!!! I was about to tell you about this `deprecation_reason` thing but you already got it!